### ISSUE TITLE
NO SOUND AFTER ENCODE!!

### ISSUE THREAD

**Issue by DominikStyp** [Anime4000/IFME#69]
I've just realized that after conversion of 10 files that NONE have a sound track.
I had set up Freware Advanced Audio Coder with bitrate 160. 
Source files are WMV with sound. Version 5.1.7.
"Unsupported output format" was outputting into the console, I don't know why.......
Video is OK but without any sound

**Comment by Anime4000** [comment]
can you try with 5.1.9?
I tell IFME, found any Windows Media format, will be force to AAC even set to pass-through.

**Comment by DominikStyp** [comment]
Just checked 5.1.9 with following settings
http://grab.by/Mk2C
File has no sound.

**Comment by Anime4000** [comment]
if WMA audio, I test using bundled files that ship with windows, is there...
![untitled - internet friendly media encoder v5 1 9 misaki meguri 64bit](https://cloud.githubusercontent.com/assets/1908715/11428998/ac1bd5ee-94ac-11e5-9281-a6d54faf917b.png)

can you run:
`ffmpeg -i file.wmv 2> video_data.txt`

let me know what inside

**Comment by DominikStyp** [comment]
if I run this command sound in output file works

ffmpeg version N-75275-gd13a2df Copyright (c) 2000-2015 the FFmpeg developers
  built with gcc 4.9.3 (GCC)
  configuration: --disable-static --enable-shared --enable-gpl --enable-version3 --disable-w32threads --enable-avisynth --enable-bzlib --enable-fontconfig --enable-frei0r --enable-gnutls --enable-iconv --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libdcadec --enable-libfreetype --enable-libgme --enable-libgsm --enable-libilbc --enable-libmodplug --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-librtmp --enable-libschroedinger --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvo-aacenc --enable-libvo-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-lzma --enable-decklink --enable-zlib
  libavutil      55.  2.100 / 55.  2.100
  libavcodec     57.  1.100 / 57.  1.100
  libavformat    57.  0.100 / 57.  0.100
  libavdevice    57.  0.100 / 57.  0.100
  libavfilter     6.  3.100 /  6.  3.100
  libswscale      4.  0.100 /  4.  0.100
  libswresample   2.  0.100 /  2.  0.100
  libpostproc    54.  0.100 / 54.  0.100
[wmv3 @ 000000000061b200] Extra data: 8 bits left, value: 0
Guessed Channel Layout for  Input Stream #0.1 : mono
Input #0, asf, from 't.wmv':
  Metadata:
    WM/ToolName     : Machete 4.1
    WMFSDKVersion   : 12.0.7601.17514
    WMFSDKNeeded    : 0.0.0.0000
    IsVBR           : 0
    DeviceConformanceTemplate: L2
  Duration: 00:00:24.02, start: 0.000000, bitrate: 3159 kb/s
    Stream #0:0(pol): Video: wmv3 (Main) (WMV3 / 0x33564D57), yuv420p, 960x540, 3112 kb/s, 25 fps, 25 tbr, 1k tbn, 1k tbc
    Stream #0:1(pol): Audio: wmav2 (a[1][0][0] / 0x0161), 22050 Hz, 1 channels, fltp, 16 kb/s
[wmv3 @ 000000000061c9e0] Extra data: 8 bits left, value: 0
Output #0, asf, to 'tt.wmv':
  Metadata:
    WM/ToolName     : Machete 4.1
    WMFSDKVersion   : 12.0.7601.17514
    WMFSDKNeeded    : 0.0.0.0000
    IsVBR           : 0
    DeviceConformanceTemplate: L2
    WM/EncodingSettings: Lavf57.0.100
    Stream #0:0(pol): Video: msmpeg4v3 (msmpeg4) (MP43 / 0x3334504D), yuv420p, 960x540, q=2-31, 200 kb/s, 25 fps, 1k tbn, 25 tbc
    Metadata:
      encoder         : Lavc57.1.100 msmpeg4
    Stream #0:1(pol): Audio: wmav2 (a[1][0][0] / 0x0161), 22050 Hz, mono, fltp, 128 kb/s
    Metadata:
      encoder         : Lavc57.1.100 wmav2
Stream mapping:
  Stream #0:0 -> #0:0 (wmv3 (native) -> msmpeg4v3 (msmpeg4))
  Stream #0:1 -> #0:1 (wmav2 (native) -> wmav2 (native))
Press [q] to stop, [?] for help
frame=  100 fps=0.0 q=31.0 size=     626kB time=00:00:04.14 bitrate=1235.9kbits/s    
frame=  202 fps=201 q=31.0 size=    1117kB time=00:00:08.14 bitrate=1123.3kbits/s    
frame=  299 fps=198 q=31.0 size=    1610kB time=00:00:12.04 bitrate=1095.2kbits/s    
frame=  407 fps=202 q=31.0 size=    2038kB time=00:00:16.55 bitrate=1009.0kbits/s    
frame=  511 fps=203 q=31.0 size=    2545kB time=00:00:20.45 bitrate=1019.3kbits/s    
frame=  600 fps=203 q=31.0 Lsize=    2964kB time=00:00:24.00 bitrate=1011.6kbits/s    
video:2499kB audio:374kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 3.146681%

**Comment by DominikStyp** [comment]
Sorry man but I lost too much time already on bugs in version 5. I'm returning to v4, it worked better. 
No I cant even convert WMV with sound, Passthrough option doesn't work either.

**Comment by DominikStyp** [comment]
I guess problem with audio encoding audio lies in Bit Rate/Level - If I set it to 192 there is no sound, but at 128 sound is back, but I want higher sound quality than 128...

**Comment by Anime4000** [comment]
* it's because version 4 and 5 work differently...
* Audio bit-rate, you need to know that your file is `16kbps` increasing makes no different

This data from Windows 7 bundled video, the Audio Id is same `wmav2 (a[1][0][0] / 0x0161)`
```
ffprobe version N-75275-gd13a2df Copyright (c) 2007-2015 the FFmpeg developers
  built with gcc 4.9.3 (GCC)
  configuration: --disable-static --enable-shared --enable-gpl --enable-version3 --disable-w32threads --enable-avisynth --enable-bzlib --enable-fontconfig --enable-frei0r --enable-gnutls --enable-iconv --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libdcadec --enable-libfreetype --enable-libgme --enable-libgsm --enable-libilbc --enable-libmodplug --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-librtmp --enable-libschroedinger --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvo-aacenc --enable-libvo-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-lzma --enable-decklink --enable-zlib
  libavutil      55.  2.100 / 55.  2.100
  libavcodec     57.  1.100 / 57.  1.100
  libavformat    57.  0.100 / 57.  0.100
  libavdevice    57.  0.100 / 57.  0.100
  libavfilter     6.  3.100 /  6.  3.100
  libswscale      4.  0.100 /  4.  0.100
  libswresample   2.  0.100 /  2.  0.100
  libpostproc    54.  0.100 / 54.  0.100
Input #0, asf, from 'C:\Users\Public\Videos\Sample Videos\Wildlife.wmv':
  Metadata:
    SfOriginalFPS   : 299700
    WMFSDKVersion   : 11.0.6001.7000
    WMFSDKNeeded    : 0.0.0.0000
    comment         : Footage: Small World Productions, Inc; Tourism New Zealand | Producer: Gary F. Spradling | Music: Steve Ball
    title           : Wildlife in HD
    copyright       : © 2008 Microsoft Corporation
    IsVBR           : 0
    DeviceConformanceTemplate: AP@L3
  Duration: 00:00:30.09, start: 0.000000, bitrate: 6977 kb/s
    Stream #0:0(eng): Audio: wmav2 (a[1][0][0] / 0x0161), 44100 Hz, 2 channels, fltp, 192 kb/s
    Stream #0:1(eng): Video: vc1 (Advanced) (WVC1 / 0x31435657), yuv420p, 1280x720, 5942 kb/s, 29.97 fps, 29.97 tbr, 1k tbn, 1k tbc
```

**Comment by Anime4000** [comment]
I test many wmv files, all work just fine @_@
I had no idea which is not working~

**Comment by SailorOnDaTea** [comment]
make sure your file us not broken. in past i used to have similar issues with Mpeg and real player formats. i also needed to fix some avi files... when stream or ccontainer is broken these videos won't work. sometimes loading via directshow helps because with avisynth it handles input differently.

**Comment by Anime4000** [comment]
FFmpeg suppose able to read all kind of format, maybe use AviSynth to by pass FFmpeg

**Comment by SailorOnDaTea** [comment]
Trying 5.1.10<br>
FLAC audio track is not copied after encode.<br>
Fonts and chapters look in tact. <br>
CLI only mentions audio track in ffmpeg input but nowhere else. it is not included in muxing or mapping later. Muxed output file has no audio track.<br>
I will fall back to 5.1.9 and report again.

![2015-12-12 21 41 41](https://cloud.githubusercontent.com/assets/10913460/11763593/92e7c5be-a119-11e5-8319-5440dbdd02d2.png)
![2015-12-12 21 43 35](https://cloud.githubusercontent.com/assets/10913460/11763594/92eba012-a119-11e5-8b4c-6a6a3705ffc7.png)

**Comment by SailorOnDaTea** [comment]
5.1.9:<br>
Audio, chapters, sub  and fonts are OK<br>
These lines about flac didn't appear on 5.1.10 CLI but show on 5.1.9:<br>
![2015-12-12 23 11 29](https://cloud.githubusercontent.com/assets/10913460/11763901/ec48add8-a125-11e5-959f-bb28ce36efd0.png)
![2015-12-12 23 12 21](https://cloud.githubusercontent.com/assets/10913460/11763902/ec544648-a125-11e5-8bb5-750b889d7843.png)

**Comment by Anime4000** [comment]
show me Audio Tab and temp folder

**Comment by Anime4000** [comment]
I don't know... I keep testing various format, this FLAC just work fine...

**Comment by Anime4000** [comment]
If got value at Audio Tab, means it's ok...
![untitled - internet friendly media encoder v5 2 archangel descending 64bit](https://cloud.githubusercontent.com/assets/1908715/11766596/0690e7b0-a1ca-11e5-91ea-cb77b28bf0ac.png)

**Comment by SailorOnDaTea** [comment]
your screenshot shows v5.2, my problem was with 5.1.10. Actually 5.1.9 also copy flac with np.

**Comment by SailorOnDaTea** [comment]
Okay, I installed 5.1.11 and loaded queue. Then I opened temp folder. This time it is AAC sound but symptoms are the same. When I click start button, 5.1.9 extracts everything in into temp folder including an audio file. But 5.1.10 and 5.1.11 starts encoding video without an audio file in temp folder.  Here are 2  screen-shots to compare.

![2015-12-14 04 23 30](https://cloud.githubusercontent.com/assets/10913460/11771579/f485b5cc-a21a-11e5-9ba8-c1fe8bc42b0e.png)
![2015-12-14 04 25 35](https://cloud.githubusercontent.com/assets/10913460/11771580/f4893f94-a21a-11e5-9a49-d00cb8e35c58.png)
